README.md

```diff
@@ -199,6 +199,8 @@ If this works, but you're still unable to load your app by scanning the QR code,
 
 If you're not able to load the `http` URL in your phone's web browser, try using the tethering/mobile hotspot feature on your phone (beware of data usage, though), connecting your computer to that WiFi network, and restarting the packager. If you are using a VPN you may need to disable it.
 
+Alternately, use the Expo XDE app, as the standard app requires a network that allows peering.
```

Comment:
That allows peering if you wish to run the app on a physical device rather than an emulator. Instead XDE connects to your phone via a tunnel over the internet, so it requires a connection to the internet.

+ a link to expo Xde